### PR TITLE
Cari Update Color

### PR DESCRIPTION
**Pull request by liu-caro**
Added the new colors from our theme and updated the homepage colors to match. 
Currently using white as the text color, but can change after more discussion.
screencapture-localhost-3000-1593841647567

### CODE REVIEW

**Review by nupur-n — APPROVED** [review]
Looks good! Just the thing I mentioned about the bottom arrow.

**Comment by liu-caro** [comment]
Thanks @nupur-n! Removed the arrow and updated the screenshot in the PR.

**Comment by Ulam22** [comment]
Could you either highlight the blue text or use a like a light blue color?

File: src/styling/projects-page-styling.css [hunk 1]
```diff
@@ -1,72 +1,71 @@
 /** Multiple card view **/
 
 .pg-container {
-    padding: 0px 30px 0px;
+  padding: 0px 30px 0px;
```
Comment (daniaalnahdi) [inline src/styling/projects-page-styling.css:4]: nitpick: remove unit when the value is 0

**Review by daniaalnahdi — APPROVED** [review]
Looks great! I added a nitpicky comment on removing the units from 0 values, but I approved since everything looks good for the most part.

File: src/styling/projects-page-styling.css (hunk 1 above)
Comment (liu-caro) [reply src/styling/projects-page-styling.css:4]: This css is for @nupur-n's page. Nupur, could you fix this in your PR?

**Pull request merged**